lib/cli/auth.rb

```diff
@@ -1,3 +1,5 @@
+require 'launchy'
```

Comment:
Shouldn't be needed. Otherwise add to  https://github.com/dblock/strava-ruby-cli/blob/master/lib/strava-ruby-cli.rb

Reply:
I got errors without the require, so I moved it to `lib/strava-ruby-cli.rb`